test/e2e/build_test.go

```diff
@@ -195,4 +195,21 @@ var _ = Describe("Podman build", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 	})
 
+	FIt("podman images --http_proxy flag", func() {
```

Comment:
Shouldn't this be 'Podman build --http-proxy flag'?

Reply:
Yes, plus the FIt is wrong.
Fixed.